managed/src/main/java/com/yugabyte/yw/models/AccessKey.java

```diff
@@ -58,13 +64,29 @@ public static AccessKey create(UUID providerUUID, String keyCode, KeyInfo keyInf
     return accessKey;
   }
 
+  public boolean delete() {
```

Comment:
Most likely this will be usuful only in that one place in the Controller. So you can just do similar logic in call site instead of adding a special method here.

Reply:
If you have a good reason to keep it then name it `deleteOrThrow` and change the return type to void since it wil always be true otherwise.